styleguide/components/31-molecules/accordion/README.md

```diff
@@ -1,22 +1,59 @@
 # Accordion
 
-## Style
+## When to use this component
 
-* Single items receive a chevron on the right hand side.
-* Multiple expandable items receive a 'plus' or 'minus' sign on the left hand side.
+An accordion component is used to:
 
-## Markup
+* Show an overview of sections of multiple, related sections of content, that can be read independently from each other and which users typically are not interested to see all, but rather one or a few specific sections.
+* Make it possible for users to show or hide those section as needed.
+* Hide a simple piece or section of content that is only necessary for a minority of the users or in special cases (progressive disclosure principle).
+
+## When not to use this component
+
+* Do not use an accordion component to hide content that is essential to all users.
+* Do not use an accordion component when the content in the items contains more than one text paragraph or also needs other components than text paragraphs to convey the message. Only use accordion components for simple content.
```

Comment:
Yes, this is always applicable. In fact, if there is only one channel, it would be better if there was no accordion (because the opening hours are expanded anyway when there is only one channel). Is this possible?

Reply:
Everything is possible, but no :)

I was actually referring to the content section:

> Do not use an accordion component when the content in the items contains more than one text paragraph or also needs other components than text paragraphs to convey the message. Only use accordion components for simple content.

Reply:
Somehow, I want to described that is it not the intention to put a lot of content inside an accordion? How would you describe this? :)

Reply:
Content that does not rely on context to be understood?

But even that doesn't really apply because we don't include channel or service information in the http://stijlgids.web.test.gentgrp.gent.be/v3/components/detail/opening-hours-accordion.html